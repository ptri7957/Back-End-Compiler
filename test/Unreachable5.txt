( (test (n)
    (0 (ld r1 n)
       (lc r2 0)
       (eq r3 r1 r2)
       (st cond r3)
       (ld r4 cond)
       (br r4 1 3) )
    (1 (lc r5 1)
       (st tmp r5)
       (ld r6 tmp)
       (ret r6) )
    (2 (ld r7 n)
       (ld r21 x)
       (lc r8 1)
       (sub r9 r1 r8)
       (st tmp r9)
       (ld r10 tmp)
       (call r11 factorial r10)
       (ld r12 n)
       (mul r13 r11 r12)
       (st tmp r13)
       (ld r14 tmp)
       (br r14 3 4) )
    (3 (ld r15 n)
       (ret r15) )
    (4 (lc r16 90)
       (st r16 x)
       (ld r17 x)
       (ret r17) ) ) )