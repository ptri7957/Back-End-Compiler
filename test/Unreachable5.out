( (test (n)
    (0 (ld r1 n)
       (lc r2 0)
       (eq r3 r1 r2)
       (st cond r3)
       (ld r4 cond)
       (br r4 1 3) )
    (1 (lc r5 1)
       (st tmp r5)
       (ld r6 tmp)
       (ret r6) )
    (3 (ld r15 n)
       (ret r15) )
    (4 (st r16 x)
       (ld r17 x)
       (ret r17) ) ) )